internal/synctest/synctest_go124.go

```diff
@@ -0,0 +1,14 @@
+//go:build go1.24 && !go1.25
+
+package synctest
+
+import (
+	"testing"
+	"testing/synctest"
+)
+
+func Test(t *testing.T, f func(t *testing.T)) {
+	synctest.Run(func() {
+		f(t)
+	})
+}
```

Comment:
For some reason, `t.Helper` doesn't seem to have any effect here. Not sure how much I care though, since this is just a shim for the old Go version anyway.